Backend start

brunofoster03 opened the pull request:
Created initial backend configuration with Node.JS and Express.JS. With app and database configuration, sequelize models and routes configuration

manduinca commented on src/models/Extra.js:
```diff
@@ -0,0 +1,32 @@
+
+// Create & export Extra model
+module.exports = (sequelize) => {
+  sequelize.define('Extra', {
```
Maybe a better name can be: `TripAdditionalCharges`

manduinca commented on src/database.js:
```diff
@@ -0,0 +1,61 @@
+const EnterpriceModel = require('./models/Enterprice.js')
+const ExtraModel = require('./models/Extra.js')
+const ReserveModel = require('./models/Reserve.js')
+const RolModel = require('./models/Rol.js')
```
Role?

manduinca commented on src/database.js:
```diff
@@ -0,0 +1,61 @@
+})
+
+// Imports models
+const EnterpriceModel = require('./models/Enterprice.js')
```
Enterprise?

manduinca commented on src/database.js:
Please review names. I think we are missing someone changes.

manduinca commented on .vscode/settings.json:
Please don't push it to repository. You can add `.vscode` line to gitignore file.

manduinca reviewed (changes requested):
I leaved some comments.

brunofoster03 committed:
Backend Init

Created backend initial configuration with index.js
Created src directory with index.js and database.js configuration
Created models directory with Enterprise, Reserve, Rol, Trip, TripAdditionalCharge and User models configuration.
Created routes directory with index.js configuration.

brunofoster03 committed:
Post authentication route

Created controllers/Auth directory with postAuth.js controller.
Created route '/auth' in routes/index.js.

manduinca commented on src/models/Rol.js:
Please don't forget this file. Role.

manduinca reviewed (changes requested):
Please change rol to role.

brunofoster03 committed:
Role model name

Changed Role model name 'Rol' to 'Role'.

manduinca commented on src/models/Rol.js:
I think you forget this file, please delete it.

brunofoster03 committed:
Delete Rol.js

Deleted Rol.js model

manduinca reviewed (approved):
LGTM!

martinalturria reviewed (approved):
LGTM
